cli/Makefile

```diff
@@ -14,7 +14,7 @@ pytest:
 	${PYTHON} -m pytest kubectl_kadalu
 
 pylint:
-	cd kubectl_kadalu && ${PYTHON} -m pylint --disable W0511 *.py
+	cd kubectl_kadalu && ${PYTHON} -m pylint --disable W0511,R0801 *.py
```

Comment:
Local disabling did not work. Testing by changing here.

Reply:
You need to fix it in top level Makefile too.

Reply:
Sure